infrastructure/k8s/services/github.yaml

```diff
@@ -0,0 +1,39 @@
+project_name: "taskcluster-github"
+secrets:
+  azure_account: ".Values.tfAzureAccount"
+  force_ssl: "false"
+  github_app_id: ".Values.githubAppId" 
+  github_private_pem: ".Values.githubPrivatePem"
+  monitoring_enable: "true"
+  node_env: "production"
+  publish_metadata: "false"
+  pulse_hostname: ".Values.pulseHostname"
+  pulse_password: ".Values.githubPulsePassword"
+  pulse_username: ".Values.githubPulseUsername"
+  pulse_vhost: ".Values.pulseVhost"
+  taskcluster_access_token: ".Values.githubAccessToken"
+  taskcluster_client_id: "static/taskcluster/github"
+  trust_proxy: "true"
+  webhook_secret: ".Values.webhookSecret"
```

Comment:
it might be clearer to name this githubWebhookSecret, since I think it's https://developer.github.com/webhooks/securing/